### PR TITLE
with this one commit

### PR DESCRIPTION
**Pull request by zrgaines**
nice

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -27,7 +27,12 @@ var list = [
 	/why we really shouldn['’]?t/i,
 	/won['’]?t believe/i,
 	/you can/i,
-	/you won['’]?t believe/i
+	/you won['’]?t believe/i,
+	/this hot new/i,
+	/signs you['`]re/i,
```
Comment (mrdoob) [inline script.js:32]: Duplicate

File: script.js [hunk 2]
```diff
@@ -27,7 +27,12 @@ var list = [
...
+	/how one man/i,
```
Comment (mrdoob) [inline script.js:33]: `/how one (man|woman)/i`

**Comment by mrdoob** [comment]
I'll merge and tweak 👌

**Comment by zrgaines** [comment]
legendary work

**Pull request merged**